vm/alpine.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+set -e
+
+sudo apt update
+sudo apt install qemu-system-x86 qemu-system-gui- qemu-system-modules-opengl- expect
+
+mkdir -p alpine && cd alpine
+
+DOWNLOAD_PATH="https://dl-cdn.alpinelinux.org/latest-stable/releases/x86"
+../download.sh $DOWNLOAD_PATH/alpine-minirootfs-3.23.2-x86.tar.gz 99457a300f13d1971d4627a0ccdf4ae17db18807804e3b5867d4edf8afa16d23
+../download.sh $DOWNLOAD_PATH/alpine-virt-3.23.2-x86.iso 4c6c76a7669c1ec55c6a7e9805b387abdd9bc5df308fd0e4a9c6e6ac028bc1cc
+
+mv -v alpine-minirootfs-*-x86.tar.gz rootfs.tar.gz
+mv -v alpine-virt-*-x86.iso virt.iso
+
+mkdir rootfs
+tar xf rootfs.tar.gz -C rootfs
+
+git bundle create rootfs/root/jou.bundle --all
+
+truncate -s 4G rootfs.img
+/sbin/mkfs.ext4 rootfs.img -d rootfs
+qemu-img convert rootfs.img -O qcow2 rootfs.qcow2
```

Comment:
Is there a reason to convert to qcow2 instead of passing the raw image to qemu directly?